packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -52,16 +52,15 @@ class AccountSignup extends React.Component {
     };
 
     onSignupComplete = error => {
+        // Handle lower level modal controls due to overriding modal rendering
+        this.props.isModalVisible(false);
+        this.props.enableApp();
+
         // Error would be returned on invalid token (and the like) cases.
         // TODO: Proper error handling (currently we have no place to put the message)
-
         if (error) {
             throw Error(error);
         }
```

Comment:
Note: I changed the ordering as the throwing the `Error` before toggling the modal visibility and enabling the app.

This is because throwing the error will cause code execution to halt before reaching the app / modal toggler.